static/app/components/events/searchBar.tsx

```diff
@@ -166,6 +167,7 @@ function SearchBar(props: SearchBarProps) {
       React.ComponentProps<typeof Measurements>['children']
     >[0]['measurements']
   ) => {
+    const transaction = Sentry.startTransaction({name: 'SearchBar.getTagList'});
```

Comment:
I'd only start the transaction the first time, put this in a useRef and just check if it already exists and don't do it again. Should be a simple way of mitigating tons of these being spawned.

Reply:
Good call, I changed to only collect this once and if tags.length > 0 to avoid collecting on page load when tags may still be loading.